css/comments.scss

```diff
@@ -8,30 +8,30 @@
  *
  */
 
-#commentsTabView .comments,
```

Comment:
Can we rename the file to chatview.scss then too?
Since it's not a copy of comments anymore?

Also if this is still a copy from the comments app, we should try to generate it from there automatically?

Reply:
>  Can we rename the file to chatview.scss then too?
>  Since it's not a copy of comments anymore?

Done.

>  Also if this is still a copy from the comments app, we should try to generate it from there automatically?

No, they are different files.